src/templates/article-latest.js

```diff
@@ -0,0 +1,565 @@
+    const side_banner_data = post_data.side_banners
+
+    const side_banner_data_details = {
+        max_w_value: '328px',
+        max_w_tablet: '320px',
+        isExternal: true,
+        redirectLink: side_banner_data.cta_url,
+        imgSrcDesktop: getImage(side_banner_data.banner_image.imageFile),
+    }
+
+    const footer_banner_details = {
+        max_w_value: '792px',
+        max_w_tablet: '580px',
+        isExternal: true,
+        redirectLink: footer_banner_data?.cta_url,
+        imgSrcDesktop: getImage(footer_banner_data?.desktop_banner_image?.imageFile),
+        imgSrcMobile: getImage(footer_banner_data?.mobile_banner_image?.imageFile),
+    }
+
+    return (
+        <Layout>
+            <SEO
+                description={localize(
+                    'Learn about the markets that you can trade online with Deriv, including forex, commodities, synthetic indices, and stock indices.',
+                )}
+                title={localize('Markets | Markets to trade | Deriv')}
+            />
+
+            <SectionContainer padding="0" position="relative">
+                <Background>
+                    <HeroContainer>
+                        <HeroLeftWrapper width="100%">
+                            <InfoText mb="16px" size="14px">
+                                {localize(convertDate(post_data?.published_date))}
+                            </InfoText>
+                            <Header as="h1" type="page-title">
+                                {post_data?.blog_title}
+                            </Header>
+                            <InfoText size="14px" mt="16px">
+                                {localize(post_data?.read_time_in_minutes + ' min read')}
+                            </InfoText>
+                            <Show.Mobile min_width="laptop">
+                                <SideBarContainer fd="column" mr="126px" height="auto">
+                                    <Flex fw="wrap" jc="flex-start" max-width="100%" width=" 100%">
+                                        {post_data?.tags.map((tag) => {
+                                            return (
+                                                <Tag key={tag.tags_id.id}>
+                                                    {tag.tags_id.tag_name}
+                                                </Tag>
+                                            )
+                                        })}
+                                    </Flex>
+                                </SideBarContainer>
+                            </Show.Mobile>
+
+                            <Show.Desktop max_width="laptop">
+                                {post_data?.author && (
+                                    <Flex ai="center" mt="40px" jc="flex-start">
+                                        <WriterImage>
+                                            <QueryImage
+                                                data={getImage(post_data?.author?.image?.imageFile)}
```

Comment:
We should be able to pass it in without `getImage` here anymore.

Reply:
Amended in latest commit